voc/python/blocks.py

```diff
@@ -204,7 +204,7 @@ def add_callable(self, function, closure=False):
                 JavaOpcodes.DUP(),
         )
 
-        self.add_str(function.name)
+        self.add_str(function.code.co_name)
```

Comment:
Looks good to me - the code object should be the canonical python name for the function.